showpass/Showpass-E2E-tests
# TA-61 Create and delete a new group on the 'Memberships' page.

**Description:**
This spec will be verifying that a new custom group can be created in React.
Steps:
1. Log into https://beta.showpass.com/manage/memberships/
as ‘[Test-user@mailinator.com](mailto:Test-user@mailinator.com)’
2. Go to Membership > Create Group
3. Populate the form using valid input and click ‘Create Group’
4. Make the group public
5. Delete just created group

**Jira ticket:**
https://showpass.atlassian.net/jira/software/projects/TA/issues/TA-61

**How to test:**
Run the following spec and ensure it passes:
cypress/e2e/admin/memberships-groups.cy.js

**Test execution results:**

<img width="896" alt="TA-61-all-tests-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/db65cf8b-c935-441c-b70f-6b751bd4f8e1">
